src/main/java/wromaciej/hvac_sim/thermo/matter/fluids/service/IdealProcess.java

```diff
@@ -0,0 +1,33 @@
+package wromaciej.hvac_sim.thermo.matter.fluids.service;
+
+import wromaciej.hvac_sim.thermo.matter.fluids.model.Fluid;
+import wromaciej.hvac_sim.thermo.matter.fluids.model.FluidType;
+import wromaciej.hvac_sim.thermo.matter.fluids.parameters.Parameter;
+
+public class IdealProcess {
+
```

Comment:
why all methods are static? You should do that as rarely as possible - it is harder to test classes that use that static functions and you cannot replace them with anything else. Use normal class methods and pass its object in dependent class cosntructor, regular DI.